monome/teletype
# Blanking screensaver with 90 min timeout

#### What does this PR do?

Blanks the Teletype display after 90 minutes of keystroke activity.  Requires a keystroke to wake the display.  That keystroke is dropped.

#### Provide links to any related discussion on [lines](https://llllllll.co/).

https://llllllll.co/t/teletype-screen-saver/9769

#### How should this be manually tested?

Set the timeout (see screensaver_mode.h) to 10 seconds, then cycle through modes, use the teletype as normal.  Every 10 seconds after your last keystroke, the display should blank.  Press any key (except for mods SHIFT, ALT, and CTRL) to wake the display.  It should reveal the last screen you were on, although it will be as if you had just navigated to the screen (input and message lines will be blank, etc).

#### Any background context you want to provide?

Static patterns on the display will cause bright cells to deteriorate more quickly than black cells.  It's somewhat healable with OLED, but would need a noisy animation to play for a while to do so.

There is another strategy to code this: by shortcutting handler_ScreenRefresh.  It would be slightly more efficient, but it won't allow the screensaver interface to be extensible.  I erred on the side of flexibility.

#### If the related Github issues aren't referenced in your commits, please link to them here.

#### I have,
